common/compositor/vk/vkshim.h

```diff
@@ -16,6 +16,7 @@
 #define VKSHIM_H_
 
 #include <drm/drm_fourcc.h>
+#include <gbm.h>
```

Comment:
> GbmToVkFormat should be replaced with DrmToVkFormat

Almost, we should replace that with a call to NativeToVkFormat. There shouldn't be any need for DrmToVkFormat or GbmToVkFormat after this conversion.